src/routes/_layout.svelte

```diff
@@ -15,5 +17,16 @@
     background-color: white;
     margin: 0 auto;
     box-sizing: border-box;
+    height: 100vh;
```

Comment:
Dain, can you explain how 100vh solves the problem? How come 100% doesn't fix the job?

Reply:
100vh is 100% of the height of space depending on the device/browser while 100% height refers to the height of the container itself.

Reply:
Got it, Dain! Thanks for the clarification